code/style.css

```diff
@@ -1,7 +1,758 @@
+  transition-duration: 4.5s;
+  visibility: hidden;
+}
+
+.black-screen__button {
+  grid-row: 3;
+  color: white;
+  font-family: 'Roboto Mono', monospace;
+  font-weight: 700;
+  font-size: 20px;
+  background: transparent;
+  width: 190px;
+  height: 43px;
+  align-self: center;
+  border: 3px solid white;
+  border-radius: 10px;
+  margin: 15px 0;
+  padding-bottom: 3px;
+  justify-self: center;
+  opacity: 0%;
+  visibility: hidden;
+  transition-duration: 4.5s;
+  transition-delay: 7s;
+}
+
+.black-screen__button:hover {
+  background-color: white;
+  color: #2D2E2F;
+  cursor: pointer;
+}
+
+.black-screen__button:active {
+  transform: translateY(4px);
+}
+
+/* Media Queries */
+
+/* Media Query Tablet */
+@media (min-width: 668px) and (max-width: 1024px) {
+
+    /* Header Section */
+    .header {
+      background-size: 1200px;
+    }
+  
+    .brand-container {
+      padding-top: 0px;
+      padding-left: 30px;
+    }
+  
+    .brand-container__brand {
+      font-size: 120px;
+    }
+  
+    /* Page */
+    .page {
+      grid-template-columns: 1fr 1fr;
+      padding: 30px 25px;
+    }
+  
+    /* Centering and Positioning Grid Items */
+    .intro {
+      display: table;
+      grid-row: 1 / span 1;
+      grid-column: 1 / span 2;
+      max-width: 700px;
+    }
+  
+    .intro__text {
+      display: table-cell;
+      vertical-align: middle;
+      text-align: center;
+      font-size: 18px;
+    }
+  
+    .faq {
+      grid-column: 1 / span 1;
+      grid-row: 2 / span 1;
+    }
+  
+    .signup {
+      margin-left: auto;
+      margin-right: auto;
+      grid-column: 2 / span 1;
+      grid-row: 2 / span 1;
+      max-height: 700px;
+    }
+
+}
+
+/* Media Query Desktop */
+
+@media (min-width: 1025px) {
+
+  /* Header Section */
+  .header {
+    height: 100vh;
+    background-size: 2600px;
+  }
+
+  .brand-container {
+    padding-top: 0px;
+    padding-left: 75px;
+  }
+
+  .brand-container__brand {
+    font-size: 170px;
+  }
+
+  /* Page */
+  .page {
+    grid-template-columns: 1fr 1fr;
+    padding: 50px 25px;
+  }
+
+  /* Centering and Positioning Grid Items */
+  .intro {
+    display: table;
+  }
+
+  .intro__text {
+    display: table-cell;
+    vertical-align: middle;
+    text-align: center;
+    font-size: 18px;
+  }
+
+  .faq {
+    grid-column: 1 / span 1;
+  }
+
+  .signup {
+    margin-left: auto;
+    margin-right: auto;
+    grid-column: 2 / span 1;
+    grid-row: 1 / span 2;
+    max-height: 600px;
+  }
+
+  .black-screen__icon {
+    max-width: 450px;
+    max-height: 450px;
+  } 
+}
+
+/* Media Query Mobile Landscape Mode */
+
+@media (max-width: 930px) and (orientation: landscape) {
+  /* Header Section */
+
+  /* Header height should be resolved by mediaQuery in JS */
+  .header {
+    max-height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+  }
+
+
+
+/* Env safe area inset left and right for safari ios when viewport-fit=cover. */
```

Comment:
How did you find all this? Looks a bit tricky  - but I guess you succeeded. No wonder you were a bit frustrated over Safari....